src/lib/atoms/KeyValue/KeyValue.tsx

```diff
@@ -35,7 +64,7 @@ function KeyValue({ label, value, className, icon, appearance, ...restProps }) {
 
             <p
                 role="heading"
-                aria-level="1"
+                aria-level={1}
```

Comment:
Why you change 
`aria-level="1"`  to   `aria-level={1}`
[link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role)

Reply:
type of props aria-level is number